pkg/gen/input/dependabot/internal/file/dependabot.go

```diff
@@ -0,0 +1,48 @@
+package file
+
+import (
+	"path/filepath"
+
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/dependabot/internal/params"
+)
+
+func NewCreateDependabotInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         filepath.Join(p.Dir, "dependabot.yml"),
+		TemplateBody: createDependabotTemplate,
+		TemplateDelims: input.InputTemplateDelims{
+			Left:  "{{",
+			Right: "}}",
+		},
+		TemplateData: map[string]interface{}{
+			"Reviewers": p.Reviewers,
+		},
+	}
+
+	return i
+}
+
+var createDependabotTemplate = `# DO NOT EDIT. Generated with:
+#
+#    devctl gen dependabot
+#
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+{{- if .Reviewers }}
+  reviewers:
+  {{- range $reviewer:= .Reviewers}} 
+  - {{$reviewer}} 
+   {{- end}}
+{{- end }}
+  ignore:
+  - dependency-name: k8s.io/*
```

Comment:
Has this rule been working somewhere? It might need to be `k8s.io/**` if not. I haven't received a response from the dependabot team about the exact supported syntax

Reply:
In our operators I did not see k8s.io PRs. What we got though was `Bump sigs.k8s.io/cluster-api from 0.3.2 to 0.3.6`, which does also not work. So we can extend this expression to `*k8s.io/*` if that works.

Reply:
In the example you provided, it is a patch version bump so should have been done anyway. But if you wanted to exclude sigs dependencies I think you'd have to add them as separate ignore items because they don't all share the same versioning. Like in your example it still would have received a PR because it is not >=0.17.0

Reply:
So IMO it doesn't make sense to exclude that here by default

Reply:
Can I do something here to merge this PR? I can't follow the conversation.

Reply:
You can keep it as is for now and move on.